README.md

```diff
@@ -26,64 +29,70 @@ Basic Concept
 Getting Started
 ---------------
 
-- You need Python3. We tested primarily with **python 3.5**
-- This was tested with a recent Ubuntu based Linux.
-
+- You need Python3.6 and above for installing tanner.
+- This was tested with a recent Ubuntu-based Linux.
 
-### Setup Redis
+### Steps to install TANNER
 
+#### Step 1: Setup Redis
 
 1. Install the Redis: ``sudo apt-get install redis-server``
-2. Start it on ``localhost`` with default ``port``: ``redis-server``
-
-### Setup PHP Sandbox
+2. Run ``redis-server`` (to start it on `localhost` with default `port`)
 
+#### Step 2: Setup PHP Sandbox
 
 1. For PHP Sandbox setup, see sandbox [manual](https://github.com/mushorg/phpox)
-2. In PHP Sandbox directory, run sandbox: ``sudo python3 sandbox.py``
+2. Go to PHP Sandbox directory: ``cd phpox``
+3. Run sandbox: ``sudo python3 sandbox.py``
```

Comment:
My suggestion is to put 3. Setup sandbox: see <link>
Because now if I follow your guide, I will go to the directory, try to run `sudo python3 sandbox.py` and it will fail with `no such file or directory`. It's confusing.

I am sorry for so many fixes, but since it's readme it's super important how it looks like :)

Reply:
In that case, I would suggest we should keep the old set of instructions for the setup of `phpox' ?

It's okay, I understand its necessary to get the README correct :)

Reply:
Yeah you can keep the old one